app/src/main/java/com/asfoundation/wallet/interact/GetDefaultWalletBalance.java

```diff
@@ -113,15 +111,13 @@ private BalanceState mapToState(Boolean enoughEther, boolean enoughTokens) {
 
   private Single<Token> getToken(String contractAddress) {
```

Comment:
If this is supposed to only return the APPC token should we do we need the contract validation here? The contract address is not even needed in the hasEnoughForTransfer method (only place using this method).

Reply:
This is ony to make sure that the address used in the purchase is appc.